fixtures/getAllWorkResponse.fixture.json

```diff
@@ -6248,7 +6248,7 @@
                       "pushedDate": "2022-04-19T17:33:23Z",
                       "author": {
                         "user": {
-                          "login": "chiedo"
+                          "login": "inkblotty"
```

Comment:
These were changed to ensure we have a test case for a person committing to someone else's PR and commenting on someone else's PR (since it's mock data, I figured it'd be ok if it's not accurate).